index.html

```diff
@@ -8,15 +8,30 @@
     <link rel="preconnect"href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Poppins:display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
     <title>Portfolio website </title>
 </head>
 <body>
     <main class="Portfolio-container">
         <header>
-            <nav>
-               <a href="#" class="logo">RIDA</a> 
-             <a href="#" class="menu-container"><img src="Images/icon-menu.png" alt="menu-icon"></a>   
-            </nav>
+          <nav>
+            <a href="#" class="logo">RIDA</a> 
+          <a href="#" class="menu-container" onclick="openNav()"><img src="Images/icon-menu.png" alt="menu-icon"></a>  
```

Comment:
## OPTIONAL 
* I am marking this as optional because the previous CR missed this but I will strongly suggest that you fix it so that your portfolio looks good. 
* On desktop device the hamburger menu shouldn't be visible